app/src/main/res/layout-sw600dp/story_fragment.xml

```diff
@@ -36,13 +37,20 @@
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator">
 
+        <asia.ivity.android.marqueeview.MarqueeView
+          android:id="@+id/marquee_view"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          marquee:speed ="10"
+          marquee:pause="500">
         <TextView
```

Comment:
The indentation seems off here now that this is a child of the marquee view--please fix. Ditto for other layout files.

Reply:
Noted.

Reply:
Still unfixed I believe.

Reply:
Noted will try with AS inbuilt reformatter.